sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -105,7 +105,11 @@ public partial class ModulesClient
     }
     public partial class StatisticsClient
     {
-        public StatisticsClient() { }
+        protected StatisticsClient() { }
+        public virtual Azure.Response<Azure.Iot.Hub.Service.Models.DevicesStatistics> GetDeviceStatistics(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Iot.Hub.Service.Models.DevicesStatistics>> GetDeviceStatisticsAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Iot.Hub.Service.Models.ServiceStatistics> GetServiceStatistics(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Why is this "System.Threading.Tasks" showing here? Why not in "using" ?

Reply:
This is the API surface that is auto generated, we don't have control over this generated file.